debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -531,15 +534,14 @@ public static SchemaNameAdjustmentMode parse(String value) {
             .withImportance(Importance.MEDIUM)
             .withDescription("The name of the data collection that is used to send signals/commands to Debezium. Signaling is disabled when not set.");
 
-    public static final Field TRANSACTION_TOPIC = Field.create("transaction.topic")
-            .withDisplayName("Transaction topic name")
+    public static final Field CUSTOM_TOPIC_NAMING_STRATEGY = Field.create("custom.topic.naming.strategy")
+            .withDisplayName("Custom topic naming strategy")
             .withGroup(Field.createGroupEntry(Field.Group.ADVANCED, 21))
             .withType(Type.STRING)
```

Comment:
Can't we use type `CLASS` and `withDefault` like in `io.debezium.connector.mysql.MySqlConnectorConfig.DATABASE_HISTORY`?